docs/theory/kinetic_model.rst

```diff
@@ -0,0 +1,124 @@
+
+* :math:`j` is the reaction :math:`j`;
+* :math:`E` is the enzyme concentration;
+* :math:`C` are the metabolite concentrations;
+* :math:`k` are the kinetic parameters;
+* :math:`\Delta G^0` is the standard Gibbs energy of reaction
+
+The rate laws can incorporate allosteric effectors, competitive inhibition, reaction mechanisms, 
+and varied protein concentrations. Common rate laws include: mass action kinetics where the rate 
+is a scaled-product of the reactant concentrations, or Michaelis-Menten kinetics which reproduce enzyme 
+saturation behavior.
+
+The current supported mechanisms are `reversible_modular_rate_law` and `irreversible_modular_rate_law` 
+<../theory/enzyme_kinetics>`, which is augmented using the Generalised Monod-Wyman-Changeux formalism for 
+allostery. Additional rate laws can be easily implemented by the user.
+
+
+Out of Scope
+============
+
+Maud's current purpose is to fit steady state data and currently isn't structured to accept anything
+other than a single point. As such, the following applications are not implemented.
+
+Dynamic model fitting
+---------------------
+
+Whilst theoretically possible and useful for circumstances such as substrate pulses, fitting concentrations
+at multiple timepoints would require a significant restructuring of the current implementation. Therefore,
+we limit measurement evalutions to steady state evaluations.
+
+Limit Cycles
+------------
+
+Parameterisations which result in limit cycles are currently unsupported. A limit cycle is a stable 
+oscillatory solution that doesn't achieve a steady state, it has been observed in Yeast glycolysis
+and can be approximated by taking the average of the ossciations as the measurement of the system, 
+as seen in `How Yeast Cells Synchronize their Glycolytic Oscillations: A Perturbation Analytic Treatment 
+<https://www.sciencedirect.com/science/article/pii/S0006349500766677>`_. However, the current method
+does not permit for averaging and will indicate that the solution is unstable, regardless of simulation time.
+
+
+Solving Systems of ODEs
+=======================
+
+To determine :math:`Sv = 0` we evaluate the system of ODEs as an initial value problem by
+integrating the system over an arbitrary period of time that is defined by the user. Tolerances for steady
+state evaluation can be altered in the source code under - src/maud/model.stan. This acts as an 
+indicator which will prompt the user if the timepoint selected is insufficient, or may be a result of Limit Cycles.
+The principle assumption of this method is that there is a unique solution for every initial point, assuming
+every :math:`x_0 \gt 0`. By initialising the concentrations as close to the measured values as possible: simulation 
+time is minimised; and of the possible non-unique solutions, the solution that is closest to the initial value and
+hence measurement will ideally be selected. There is no current implementation in Maud to account for multiple solutions.
```

Comment:
Maybe rephrase to "no _other_ implementation" as initialising near the measurements is kind of a way of accounting for them.